presto-main-base/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -1913,4 +1918,70 @@ public void testInvalidTemporaryFunctionName()
         assertFails(INVALID_FUNCTION_NAME, "CREATE TEMPORARY FUNCTION sum() RETURNS INT RETURN 1");
         assertFails(INVALID_FUNCTION_NAME, "CREATE TEMPORARY FUNCTION dev.test.foo() RETURNS INT RETURN 1");
     }
+
+    @Test
+    public void testTableFunctionNotFound()
```

Comment:
Can you please add some tests for 
1.nested table function calls (calling same function different levels underneath) as well as different table functions.
2. Table functions in different operations like filter / joins/(agg if supported) ctes, 


eg
```

SELECT t1.*, tvf.col 
FROM some_table t1 
JOIN TABLE(system.simple_table_function('join_col')) tvf ON t1.id = tvf.col;
```
```

SELECT * FROM some_table 
WHERE id IN (
    SELECT filter_col FROM TABLE(system.simple_table_function('in_col'))
    WHERE filter_col > 0
);
```

etc etc

Reply:
@jaystarshot : The current PRs are just the start of simple Table functions... so only scalar arguments are added so far. Table arguments aren't supported yet.. They will start coming in subsequent PRs. So would be great to show these cases then.